packages/patternfly-4/react-charts/src/components/ChartTheme/ChartTheme.d.ts

```diff
@@ -1,15 +1,19 @@
-export const ChartBaseTheme = any;
-export const ChartDonutTheme = any;
+import { VictoryThemeInterface } from "victory";
 
-export interface ChartThemeColor {
+interface ChartBaseThemeInterface extends VictoryThemeInterface {}
+interface ChartDonutThemeInterface extends VictoryThemeInterface {}
```

Comment:
Believe this should be VictoryThemeDefinition?

I wasn't using this interface because that is only a subset of the possible charts. Was thinking we needed to create our own? I encountered a type issue while trying to use VictoryThemeDefinition with my own custom theme in Cost Management -- had to use type any for now.

![Screen Shot 2019-05-22 at 3 45 41 PM](https://user-images.githubusercontent.com/17481322/58203898-27c43c80-7ca9-11e9-82fb-ed2f4470adbb.png)

Everything else looks fine.

Reply:
Updated